docs/integrations/metrics.md

```diff
@@ -0,0 +1,3 @@
+# Metrics 
+
+`trivy-operator` exposed a `/metrics` endpoint by default  with metrics for vulnerabilities, exposed secrets, and configaudits.
```

Comment:
We must not forget that the metrics endpoint also expose interesting operator-metrics provided by controller-runtime.